src/wasmtime/crates/lind-multi-process/src/lib.rs

```diff
@@ -21,6 +21,8 @@ const ASYNCIFY_STOP_UNWIND: &str = "asyncify_stop_unwind";
 const ASYNCIFY_START_REWIND: &str = "asyncify_start_rewind";
 const ASYNCIFY_STOP_REWIND: &str = "asyncify_stop_rewind";
 
+const LIND_FS_ROOT: &str = "/home/lind-wasm/src/RawPOSIX/tmp";
```

Comment:
should root be tmp here?

Reply:
This is the root set in RawPOSIX
https://github.com/Lind-Project/lind-wasm/blob/57876e5c51e78c9285fa6842d5e1efe5f15c745f/src/RawPOSIX/src/safeposix/syscalls/fs_calls.rs#L31
I just used the same path here

Reply:
im curious that why we need using lind fs in wasmtime..?

Reply:
it is used in execve. The path passed in here should be within lind filesystem